test/Microsoft.SqlTools.ServiceLayer.Test.Common/TestData/ObjectExplorer/Baselines/AllSqlObjects.txt

```diff
@@ -0,0 +1,136 @@
+NodeType: Table Label: HumanResources.Employee
```

Comment:
That's exactly what I did here, and I think OE is the only that's using adventure works